### ISSUE TITLE
Formatting bug when using Week of Year in template

### ISSUE THREAD

**Issue by muness** [liamcain/obsidian-periodic-notes#33]
For context, I am trying to add a link from daily notes back to the corresponding weekly note.

## Observed behavior:

```
[[{{date:gggg}}
-W{{date:ww}}]]

[[{{date:gggg}}-W{{date:ww}}]]
```

Renders as

```
[[2021
-W14]]

[[2021}}-13{{3amt3:14]]
```

## Desired behavior

```
[[{{date:gggg}}-W{{date:ww}}]]
```
should render as
```
[[2021-W14]]
```

**Comment by muness** [comment]
Another example as I was trying to simplify the test case:

```
{{date:YYYY-MM}} {{date:YYYY-MM}}
```

```
2021-03}} {{3amt3:2021-03
```

So it's something about two `date` templates on the same line, maybe?

**Comment by liamcain** [comment]
Hi @muness, thanks for the report. Looks like it's a regression I introduced with the latest version. I can reproduce the issue and will have a fix shortly.

**Comment by liamcain** [comment]
This should be fixed in the new version 0.0.13. Let me know if you're still having issues.

---

Also, unrelated, but instead of `[[{{date:gggg}}-W{{date:ww}}]]`, you can just do `[[{{date:gggg-[W]ww}}]]` for something a bit simpler 😄

**Comment by muness** [comment]
@liamcain thanks for the quick fix! 

believe me, it started with the `[[{{date:gggg-[W]ww}}]]` and got increasingly more complicated as I was trying to figure out what I was doing wrong and then realizing there was probably a regression to report. Went back to it after updating and it's working as expected again. Yay!
